zulip_bots/zulip_bots/bots/incrementor/incrementor.py

```diff
@@ -1,31 +1,33 @@
 # See readme.md for instructions on running this code.
 
+from zulip_bots.lib import ExternalBotHandler
```

Comment:
I'm not sure, but I believe the idea is to leave these as Any for now, pending a general interface class.
@showell ?

Reply:
yep, `Any` is good for now